app/services/folio/locations.rb

```diff
@@ -13,6 +13,53 @@ def self.label(code:)
       instance.data.dig(code, 'discoveryDisplayName')
     end
 
+    # Output a CSV of all locations, flattening objects and converting UUIDs to codes
+    # @return [String] comma-separated data for all locations
+    def self.to_csv
+      headers = %w[institutionCode campusCode libraryCode primaryServicePointCode servicePointCodes]
```

Comment:
Just thinking about how people would use this spreadsheet, I'm not sure we want primaryServicePointCode + servicePointCodes at the beginning of the list. At the very least, I think the location code and name would warrant a top spot (for "freezing" those first columns, if nothing else)